src/Polysemy/Output.hs

```diff
@@ -116,6 +118,44 @@ runOutputMonoidTVar tvar f = interpret $ \case
     writeTVar tvar $! s <> f o
 {-# INLINE runOutputMonoidTVar #-}
 
+
+--------------------------------------------------------------------
+-- | Run an 'Output' effect in terms of atomic operations
+-- in 'IO'.
+--
+-- Internally, this simply creates a new 'IORef', passes it to
+-- 'runOutputMonoidIORef', and then returns the result and the final value
+-- of the 'IORef'.
+outputToIOMonoid
```

Comment:
Since `outputToIOMonoid` uses an `IORef`, it means it doesn't respect local/global state semantics depending on the order of interpreters run. Should a _Beware_ be added for that?

Reply:
Done